demos/keypoints_bounding_boxes/src/demo.py

```diff
@@ -164,19 +148,18 @@ def keypoints_distance(detected_pose, tracked_pose):
     # CLI configuration
     parser = argparse.ArgumentParser(description="Track objects in a video.")
     parser.add_argument("files", type=str, nargs="+", help="Video files to process")
-    parser.add_argument("--detector_path", type=str, default="yolov5m6.pt", help="YOLOv5 model path")
-    parser.add_argument("--img_size", type=int, default="720", help="YOLOv5 inference size (pixels)")
-    parser.add_argument("--conf_thres", type=float, default="0.25", help="YOLOv5 object confidence threshold")
-    parser.add_argument("--iou_thresh", type=float, default="0.45", help="YOLOv5 IOU threshold for NMS")
+    parser.add_argument("--model-name", type=str, default="yolov5m6", help="YOLOv5 model name")
+    parser.add_argument("--img-size", type=int, default="720", help="YOLOv5 inference size (pixels)")
+    parser.add_argument("--conf-thres", type=float, default="0.25", help="YOLOv5 object confidence threshold")
+    parser.add_argument("--iou-thresh", type=float, default="0.45", help="YOLOv5 IOU threshold for NMS")
```

Comment:
We are using "thres" and "thresh" inconsistently.

Reply:
Good catch! I will check the name of the arguments in all of the demos and fix them